recipes/natural_language_processing/chatbot/app/requirements.txt

```diff
@@ -1,3 +1,4 @@
-langchain==0.1.20
+langchain==0.2.3
```

Comment:
Please be aware the renovate is configured to allow versions lower then 0.2.2. Maybe renovate restriction should be removed if that is no longer needed.